remove fake 'n' section in manpage, remplace it by section '3' (libraries)
--- /dev/null
+++ b/src/man/mann/ow.3
@@ -0,0 +1,2 @@
+.so man3/owtcl.3
+
--- a/src/man/mann/Makefile.am
+++ b/src/man/mann/Makefile.am
@@ -5,24 +5,24 @@
 SOFILES = \
     description.nso seealso.nso
 ## man files that need no preprocessing
-dist_mann_MANS = \
-    ow.n
+dist_man3_MANS = \
+    ow.3
 ## file to include in distribution
 EXTRA_DIST = $(SOFILES) $(MANFILES)
 
 
 if SOELIM
 
-mann_MANS = $(addsuffix .n,$(basename $(MANFILES)))
-CLEANFILES = $(mann_MANS)
+man3_MANS = $(addsuffix .3,$(basename $(MANFILES)))
+CLEANFILES = $(man3_MANS)
 
 # preproc man pages via soelim
-$(mann_MANS): $(MANFILES) $(SOFILES)
-%.n :: %.man
+$(man3_MANS): $(MANFILES) $(SOFILES)
+%.3 :: %.man
 	$(SOELIM) -r -I $(srcdir)/.. $< > $@
 
 else !SOELIM
 
-mann_MANS = $(MANFILES) $(SOFILES)
+man3_MANS = $(MANFILES) $(SOFILES)
 
 endif !SOELIM
--- a/src/man/mann/ow.n
+++ /dev/null
@@ -1 +0,0 @@
-.so mann/owtcl.n
--- a/src/man/mann/owtcl.man
+++ b/src/man/mann/owtcl.man
@@ -231,8 +231,7 @@
 .de UL
 \\$1\l'|0\(ul'\\$2
 ..
-.TH "Owtcl" TCL "20/January/2005" "Tcl"
-.HS table tk
+.TH OWTCL 3 2005-01-20 "Tcl"
 .BS
 .SH NAME
 Owtcl \- OWFS library access commands for Tcl
